Tribler/Core/Modules/restapi/my_channel_endpoint.py

```diff
@@ -144,6 +142,37 @@ def render_PUT(self, request):
             request.setResponseCode(http.BAD_REQUEST)
             return json.dumps({"error": "rss_feed_url parameter missing"})
 
-        channel_obj.create_rss_feed(parameters['rss_feed_url'][0])
+        rss_feed_url = parameters['rss_feed_url'][0]
+
+        if add:
```

Comment:
Having a function that does completely opposite things depending on an argument is not really nice. Why don't you move the code in the if/else block to each `render_*` handler instead?